xlml/apis/task.py

```diff
@@ -352,7 +352,7 @@ def run_with_run_name_generation(self) -> DAGNode:
       )
 
       # Set run_name in run_model_cmds
-      new_run_model_cmds = [f"export M_RUN_NAME={run_name}"]
+      new_run_model_cmds = [f"export RUN_NAME={run_name}"]
```

Comment:
This will only work for scripts that explicitly set `run_name=$RUN_NAME`. Could you revert this back and then change the scripts to do this https://github.com/google/maxtext/blob/main/end_to_end/test_convergence_1b_params.sh#L25-L28?

Reply:
Thank you @raymondzouu.
Updated as recommended.